inspirehep/modules/records/tasks.py

```diff
@@ -204,3 +210,38 @@ def actions():
         'success': success,
         'failures': [failure for failure in failures or []],
     }
+
+
+@shared_task(ignore_result=False, bind=True)
+def index_new_cited_records(self, pid_type, pid_value, db_version):
```

Comment:
I hate to say this, but I think the name is not very clear, as you also index records that were cited but are not anymore. Maybe `index_modified_citations_from_record`? I would also add a docstring for good measure :)